<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
color: rgb(91, 143, 155);
font-weight:600;
background-color: rgb(162, 192, 204);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>10</y>
      <width>184</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   	border: 2px solid rgb(255,255,244);
	background-color:  rgb(255,255,244);
	color: rgb(91, 143, 155);

}</string>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="glView" name="view" native="true">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>8</y>
      <width>585</width>
      <height>702</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rotate">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>60</y>
      <width>184</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="translation">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>120</y>
      <width>184</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Translation</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>190</y>
      <width>184</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Scaling</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotation_xy">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotation_xz">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotation_yz">
    <property name="geometry">
     <rect>
      <x>744</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="trans_x">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>150</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="trans_y">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>150</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="trans_z">
    <property name="geometry">
     <rect>
      <x>744</x>
      <y>150</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>220</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="custom_edges">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>290</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="type">
     <attribute name="title">
      <string>Type</string>
     </attribute>
     <widget class="QRadioButton" name="solid_edge">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Solid</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="dashed_edge">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dashed</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="thikness_edge">
     <attribute name="title">
      <string>Thickness</string>
     </attribute>
     <widget class="QSpinBox" name="thickness_edge">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>20</y>
        <width>150</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>260</y>
      <width>184</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Custom edges</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>398</y>
      <width>184</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Custom vertices</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="custom_vertices">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>423</y>
      <width>201</width>
      <height>80</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="merhod_vert">
     <attribute name="title">
      <string>Method</string>
     </attribute>
     <widget class="QRadioButton" name="none_point">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>15</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="circle_point">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>15</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Circle</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="square_point">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>15</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Square</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="size_vert">
     <attribute name="title">
      <string>Size</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="size_point">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>521</y>
      <width>184</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Set colors</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="colorback">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>550</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

  	border: 2px solid rgb(255,255,244);
	background-color:  rgb(255,255,244);
    border-width: 2px;
    border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Color of Backround</string>
    </property>
   </widget>
   <widget class="QPushButton" name="coloredge">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>577</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  	border: 2px solid rgb(255,255,244);
	background-color:  rgb(255,255,244);
    border-width: 2px;
    border-radius: 10px;

}</string>
    </property>
    <property name="text">
     <string>Color of Edges</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorvert">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>604</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  	border: 2px solid rgb(255,255,244);
	background-color:  rgb(255,255,244);
    border-width: 2px;
    border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Color of  Vertices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="parallel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>667</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  	border: 2px solid rgb(255,255,244);
	background-color:  rgb(255,255,244);
    border-width: 2px;
    border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>Parallel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>642</y>
      <width>184</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Custom vertices</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="central">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>692</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

  	border: 2px solid rgb(255,255,244);
	background-color:  rgb(255,255,244);
    border-width: 2px;
    border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>Central</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glView</class>
   <extends>QWidget</extends>
   <header location="global">glview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
